tensorboard/webapp/types/data.ts

```diff
@@ -22,8 +22,17 @@ export enum DataLoadState {
   FAILED,
 }
 
+export enum LoadFailureCode {
+  UNKNOWN = 'UNKNOWN',
```

Comment:
Just wondering, are you using the value of these enum in any meaningful way? If not, omit it so they remain as number for smaller bundle.